### PR TITLE
MNT: add maintainer

### PR DESCRIPTION
**Pull request by MementoRC**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by MementoRC** [comment]
@henryiii please acknowledge being a maintainer here

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ source:
 build:
   noarch: python
   script: {{ PYTHON }} -m pip install . -vvv
-  number: 0
+  number: 1
```
Comment (henryiii) [inline recipe/meta.yaml:15]:
```suggestion
  number: 0
```

**Comment by henryiii** [comment]
@conda-forge-admin, please rerender

**Comment by henryiii** [comment]
Sure

File: recipe/meta.yaml (hunk 1 above)
Comment (MementoRC) [reply recipe/meta.yaml:15]: Oh ... silly me. LL

**Pull request merged**